doc/news/_preparation_next_release.md

```diff
@@ -171,7 +198,7 @@ compiled against an older 0.8 version of Elektra will continue to work
 
 ### Core
 
-- <<TODO>>
+- `kdbGet` now calls global postgetstorage plugins with the parent key passed to `kdbGet`, instead of a random mountpoint. _(Klemens Böswirth)_
```

Comment:
👍

Reply:
One thing I noticed (but didn't look into): At least some errors returned from global plugins are suppressed by `kdbGet`. You may want to look into that while working on #2457.

Reply:
Yep, we didn't even use return values until #2307. The global plugin implementation needs to be re-worked and tested properly, so that it matches the proposal.